Resources/Prototypes/SS220/Loadouts/loadout_groups.yml

```
@@ -139,30 +139,64 @@
   # Адвокат (Конец)
 
   # Офицер "Синий Щит" (Начало)
+- type: loadoutGroup
+  id: BlueShieldHead
+  name: loadout-group-blueshield-head
+  minLimit: 0
+  loadouts:
+  - BlueshieldBeret
+  - BlueshieldBlueBeret
+
+- type: loadoutGroup
+  id: BlueShieldGlasses
+  name: loadout-group-blueshield-glasses
+  minLimit: 0
```

Я бы 1 сделал, чтобы щит без очков не мог появиться даже по случайности
```suggestion
  minLimit: 1
```